dockerfiles/docker-compose.yml

```diff
@@ -61,6 +63,8 @@ services:
     environment:
       - INIT=${INIT:-}
       - DOCKER_NO_RELOAD
+      - RTD_EXT_THEME_ENABLED
+      - RTD_EXT_THEME_DEV_SERVER_ENABLED
```

Comment:
We should put a default value here, if we don't pass this env var docker will raise a warning.

Reply:
I haven't noticed this at all, and do not have the env variables set. I was able to bring up the instance without webpack/theme -- that is, these env variables were not set

Reply:
I've held off on this because it requires more changes to rtd settings. If anyone notices any issues with warnings here, we can revisit, but I have not encountered any warnings about the missing variables.